python/llm/example/GPU/Deepspeed-AutoTP/run_qwen_14b_arc_2_card.sh

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+## Validated BKC for Qwen1.5-14B-Chat on 2 ARC with
+## Ubuntu 22.04.4, kernel 6.5.0-27-generic, level-zero 1.14.0, NEO(compute runtime) 24.09.28717.12
```

Comment:
Any guide how to install level-zero and NEO? Do the users need to download the deb files or can install using apt?